include/benchmark/benchmark.h

```diff
@@ -1,386 +1,48 @@
-// Support for registering benchmarks for functions.
-
-/* Example usage:
-// Define a function that executes the code to be measured a
-// specified number of times:
-static void BM_StringCreation(benchmark::State& state) {
-  while (state.KeepRunning())
-    std::string empty_string;
-}
+// Copyright 2015 Google Inc. All rights reserved.
```

Comment:
any chance you can move this change to another patch? it's making it harder to review the deltas between the benchmark API.